examples/medplum-fhircast-demo/src/index.css

```diff
@@ -0,0 +1,77 @@
+:root {
```

Comment:
Probably out of scope for this PR, but for the future:  When we add auth to everything, we'll probably want to use the `@medplum/react` React components, especially for `<SignInForm>` and some of the auth helper components.  Once we break the seal and start using Mantine, we might as well use it for all base styling.

We can discuss that offline.